check_upgrade.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+VERSION=3.1.0.1971
+
+# stop current cluster
+./start-splice-cluster -k
+
+# clean up platform_it
+cd platform_it
+git clean -dfx
+cd ..
+
+# download the previous standalone data
+aws s3 cp s3://splice-snapshots/upgrade_tests/platform_it_${VERSION}.tar.gz .
+tar -xzvf platform_it_${VERSION}.tar.gz
+rm platform_it_${VERSION}.tar.gz
+
+# restart cluster
+./start-splice-cluster -l
+
+# test
+if mvn -B -e surefire:test -Pcore,cdh6.3.0 -Dtest='UpgradeTestIT#*' -DskipServerStart -DfailIfNoTests=false; then
```

Comment:
Is `mvn` return code reliable here?

Reply:
yes, returning non-null in case of failed test